masterlist.yaml

```diff
@@ -8820,6 +8820,19 @@ plugins:
         util: '[SSEEdit v4.0.1](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
         itm: 773
 
+  - name: 'WoodworkersWhim.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/64507'
+        name: 'Nexus Mods'
+    group: *lateChangesGroup
+    clean:
+      # Normal Plugin
+      - crc: 0x24E8B4EA
+        util: '[SSEEdit v4.0.1](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
```

Comment:
My bad on that. I copy-pasted from the one above to get the link, but it looks like I forgot to check the versioning. And it would seem that my efforts were in vain anyway.